### ISSUE TITLE
HTTP 500 errors trying to get to schedules after creating a schedule using the new prompt function

### ISSUE THREAD

**Issue by grahamn-gr** [ansible/awx#1260]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - API
 - UI

##### SUMMARY
HTTP 500 errors trying to get to schedules after creating a schedule using the new prompt function

##### ENVIRONMENT
* AWX version: 1.0.3.36
* AWX install method: docker on linux
* Ansible version:  2.4.3.0

##### STEPS TO REPRODUCE

1) Create template that has a survey
2) Schedule a job against the template and use the blue prompt button to fill in the survey
3) Click on the calendar icon on the main navigation bar, or click on jobs and schedules

##### EXPECTED RESULTS

You're able to see all schedules

##### ACTUAL RESULTS

Error! Invalid search term entered. GET returned: 500 A server error has occurred.

**Comment by grahamn-gr** [comment]
Logs from awx_web

```
2018-02-16 09:03:42,889 ERROR    django.request Internal Server Error: /api/v2/schedules/
Traceback (most recent call last):
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/utils/decorators.py", line 185, in inner
    return func(*args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/lib/python2.7/site-packages/awx/api/generics.py", line 255, in dispatch
    return super(APIView, self).dispatch(request, *args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/views.py", line 489, in dispatch
    response = self.handle_exception(exc)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/views.py", line 449, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/views.py", line 486, in dispatch
    response = handler(request, *args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/generics.py", line 201, in get
    return self.list(request, *args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/mixins.py", line 42, in list
    page = self.paginate_queryset(queryset)
  File "/usr/lib/python2.7/site-packages/awx/api/generics.py", line 331, in paginate_queryset
    page = super(ListAPIView, self).paginate_queryset(queryset)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/generics.py", line 173, in paginate_queryset
    return self.paginator.paginate_queryset(queryset, self.request, view=self)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/pagination.py", line 214, in paginate_queryset
    self.page = paginator.page(page_number)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/paginator.py", line 57, in page
    number = self.validate_number(number)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/paginator.py", line 46, in validate_number
    if number > self.num_pages:
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/paginator.py", line 91, in num_pages
    if self.count == 0 and not self.allow_empty_first_page:
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/paginator.py", line 79, in count
    return self.object_list.count()
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/db/models/query.py", line 364, in count
    return self.query.get_count(using=self.db)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/db/models/sql/query.py", line 499, in get_count
    number = obj.get_aggregation(using, ['__count'])['__count']
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/db/models/sql/query.py", line 480, in get_aggregation
    result = compiler.execute_sql(SINGLE)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 894, in execute_sql
    raise original_exception
ProgrammingError: more than one row returned by a subquery used as an expression

2018-02-16 09:03:42,889 ERROR    django.request Internal Server Error: /api/v2/schedules/
Traceback (most recent call last):
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/utils/decorators.py", line 185, in inner
    return func(*args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/lib/python2.7/site-packages/awx/api/generics.py", line 255, in dispatch
    return super(APIView, self).dispatch(request, *args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/views.py", line 489, in dispatch
    response = self.handle_exception(exc)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/views.py", line 449, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/views.py", line 486, in dispatch
    response = handler(request, *args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/generics.py", line 201, in get
    return self.list(request, *args, **kwargs)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/mixins.py", line 42, in list
    page = self.paginate_queryset(queryset)
  File "/usr/lib/python2.7/site-packages/awx/api/generics.py", line 331, in paginate_queryset
    page = super(ListAPIView, self).paginate_queryset(queryset)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/generics.py", line 173, in paginate_queryset
    return self.paginator.paginate_queryset(queryset, self.request, view=self)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/rest_framework/pagination.py", line 214, in paginate_queryset
    self.page = paginator.page(page_number)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/paginator.py", line 57, in page
    number = self.validate_number(number)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/paginator.py", line 46, in validate_number
    if number > self.num_pages:
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/paginator.py", line 91, in num_pages
    if self.count == 0 and not self.allow_empty_first_page:
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/core/paginator.py", line 79, in count
    return self.object_list.count()
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/db/models/query.py", line 364, in count
    return self.query.get_count(using=self.db)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/db/models/sql/query.py", line 499, in get_count
    number = obj.get_aggregation(using, ['__count'])['__count']
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/db/models/sql/query.py", line 480, in get_aggregation
    result = compiler.execute_sql(SINGLE)
  File "/var/lib/awx/venv/awx/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 894, in execute_sql
    raise original_exception
ProgrammingError: more than one row returned by a subquery used as an expression
```

**Comment by AlanCoding** [comment]
I can't reproduce this following the steps that you gave (no 500 error). Could you give any more details about what type of survey questions you saved, or anything else that might make a difference?

**Comment by AlanCoding** [comment]
oh, it looks like this was fixed in #802, closing for now.
